src/recipes.jl

```diff
@@ -174,6 +174,18 @@ end
 end
 @deps scatterpath path scatter
 
+# ---------------------------------------------------------------------------
+# pixel
+
+@recipe function f(::Type{Val{:pixel}}, x, y, z)
```

Comment:
I don't think this recipe is needed.
Anyways, you can't return the same seriestype in a recipe with which you entered.
I'd recommend to have a look at daniels blog post: https://daschw.github.io/recipes/#how_do_recipes_actually_work

Reply:
Thanks @BeastyBlacksmith :innocent: , I created a recipe as per the discussion with @isentropic.
I will be happy to hear new ideas. :smile:

Reply:
you can just delete it, should work without it

Reply:
`:pixel` is a "Atomic" seriestype so to speak

Reply:
do not we want to fallback to small markersizes if a backend can't support it?

Reply:
I doubt that all backends can support this natively

Reply:
I will check it for each backend and then I will make the changes accordingly.

Reply:
in that case the output seriestype should be `:scatter`

Reply:
right, it should fallback to scatter